packages/docs/site/docs/01-start-here/03-repos-overview.md

```diff
@@ -0,0 +1,79 @@
+
+## [WordPress Playground](https://github.com/WordPress/wordpress-playground/)
+This is the main repository for the WordPress Playground project. It contains the core codebase for the project, project boards and issues, and discussions around ongoing project improvements and maintenance.
+
+The repo is there to guide the core contributors to the Playground identify issues, feature requests, and other improvements that need to be made to the project.
+
+Opening issues in the WordPress Playground repository for the "online" version of the Playground, IE: playground.wordpress.net, is a great way to get visibility on a problem or feature request. But there may be a better suited repo below to open the issue, and we'll discuss that shortly.
+
+Searching for issues is a great way to find a fix or workaround, or to see if a feature request has already been made. If you find an issue that matches your problem, you can add a comment to the issue to provide more information or to show that you are also experiencing the same problem.
+
+Explore the WordPress Playground [repository](https://github.com/WordPress/wordpress-playground/)
+
+---
+
+## [Playground Tools]((https://github.com/WordPress/playground-tools/))
+This repository contains a collection of tools and utilities that are used to manage and maintain the WordPress Playground project. The tools include scripts for building and deploying the project, as well as utilities for managing the documentation and other project assets.
+
+There are also advanced administration features, integrations such as the Playground Plugin, Blueprint Builder, and Interactive Code Block that live in the Playground Tools repository.
+
+This repository is also the best place to search for issues related to `wp-now` or the `VS Code Extension` for WordPress Playground. If the request you are looking for is not specific to the online version of the Playground, it may be in the Playground Tools repository.
+
+#### Tools that exist in the Playground Tools repository:
+- [Blueprint Builder](https://playground.wordpress.net/builder/builder.html) - a tool to create a view Blueprints side-by-side
+- [Playground Block](https://wordpress.org/plugins/interactive-code-block/) - embed Playground via a block on your WordPress site
+- [Interactive Code Block](https://wordpress.org/plugins/interactive-code-block/) - an interactive code editor to demonstrate and teach your readers how WordPress plugins are built.
+- [Playground Plugin](https://wordpress.org/plugins/playground/) - Copy your site or test plugins easily
+- [VS Code Extension](https://marketplace.visualstudio.com/items?itemName=WordPressPlayground.wordpress-playground) - Easily spin up WordPress instances in VS Code
+- [`wp-now` ](https://www.npmjs.com/package/%40wp-now/wp-now)- a fully featured, extensible environment for WordPress development
+- [Pull Request Previewer](https://playground.wordpress.net/wordpress.html) - a tool to preview pull requests in the Playground
```

Comment:
Nitpick: This one is actually in the main Playground repo 😅 I'd love to move it over, just didn't get to it yet

Reply:
It probably is helpful to actually describe what tools are in where, so devs don't have to go looking around everywhere. Copilot wrote most of the tools stuff and I filled in the details, didn't actually go digging in the repos 😆

Reply:
PR Previewer is in the Playground repo

Will update

Reply:
Good call. It's good for outlines and such, but it important to put the personal love into it.